### PR TITLE
Samples: Increase the resources so that the examples can run successfully

### PR DESCRIPTION
**Pull request by haoxins**
The current samples can't submit successfully because the memory exceeded the default limitation.
So that I increased the resources to make the samples can submit and run successfully.

### CODE REVIEW

File: config/samples/flinkoperator_v1beta1_flinkjobcluster.yaml [hunk 1]
```diff
@@ -25,18 +25,19 @@ spec:
...
+        memory: "2048Mi"
+        cpu: "500m"
   job:
     jarFile: ./examples/streaming/WordCount.jar
     className: org.apache.flink.streaming.examples.wordcount.WordCount
     args: ["--input", "./README.txt"]
     parallelism: 2
   flinkProperties:
     taskmanager.numberOfTaskSlots: "1"
+    taskmanager.memory.flink.size: "1024m"
```
Comment (regadas) [inline config/samples/flinkoperator_v1beta1_flinkjobcluster.yaml:43]: setting this is not required; for Flink >= v1.10 users should set `taskmanager.memory.process.size` but this setting is automatically derived based on `MemoryProcessRatio` which defaults to 90

**Review by regadas — COMMENTED** [review]
Thanks for this @haoxins !

+1 on increasing the resource limits; left a comment on the taskamanager memory as it's not required for the example.

File: config/samples/flinkoperator_v1beta1_flinkjobcluster.yaml (hunk 1 above)
Comment (haoxins) [reply config/samples/flinkoperator_v1beta1_flinkjobcluster.yaml:43]: I test with Flink `1.14` and add this prop fix the issue.
`taskmanager.memory.process.size` not works

Comment (haoxins) [reply config/samples/flinkoperator_v1beta1_flinkjobcluster.yaml]: Yeah, removed these props.
And I have verified that increase the resources make all thinks works~

**Comment by haoxins** [comment]
> Thanks for this @haoxins !
> 
> +1 on increasing the resource limits; left a comment on the taskamanager memory as it's not required for the example.

Fixed~

**Review by regadas — APPROVED** [review]
Thanks for doing this @haoxins

**Pull request merged**